packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/InterpolatedText.java

```diff
@@ -0,0 +1,18 @@
+package io.flutter.plugins.googlemaps;
+
+public class InterpolatedText {
+    String startText;
+    String endText;
+    float step;
+
+    InterpolatedText(String startText, String endText, float step) {
+        this.startText = startText;
+        this.endText = endText;
+        this.step = step;
+    }
+
+    @Override
+    public String toString() {
+        return startText + " " + endText + " " + step;
+    }
+}
```

Comment:
```suggestion
}

```
Just adding the breakline (Github suggestion)

Reply:
Actually I will just remove this class (not needed anymore)